src/Games/NexusMods.Games.StardewValley/Installers/SMAPIInstaller.cs

```diff
@@ -120,6 +120,7 @@ public class SMAPIInstaller : AModInstaller
                 installDataFile.Item.StreamFactory!,
                 (tx, id) => 
                     tx.Add(id, FilePathMetadata.OriginalName, installDataFile.Item.FileName),
+                "SMAPI: Stardew Modding API",
                 cancellationToken);
```

Comment:
This shouldn't really have a mod name. This isn't a mod, this is just a generic archive that we need to extract.